data-es6.js

```diff
@@ -66,12 +66,10 @@ exports.browsers = {
     short: 'IE 11',
     obsolete: false
   },
-  ie11tp: {
-    full: 'Internet Explorer',
-    short: 'IE<br>Technical<br>Preview',
-    unstable: true,
-    note_id: 'ie-experimental-flag',
-    note_html: 'Have to be enabled via "Experimental Web Platform Features" flag'
+  projectspartan: {
+    full: 'Project Spartan',
```

Comment:
Spartan is fine and I will submit a new PR when the new name will be announced :)